waspc/data/Generator/templates/server/src/actions/types.ts

```diff
@@ -0,0 +1,33 @@
+{{={= =}=}}
+import {
+  {=# allEntities =}
+  {= . =},
+  {=/ allEntities =}
+} from '../entities'
+
+import {
+  {=^ allUseAuth =}
```

Comment:
As @shayneczyzewski commented, it is true that it is hard to see this `^` here, and even when we see it, it is not easy to remember waht it does.
One solution might be trying to avoid it -> in this case we could pass notAllUseAuth instead of allUseAuth and that would solve it. But I am also ok with leaving it as it is.

Reply:
I think I got something better.

Reply:
Fixed [here](https://github.com/wasp-lang/wasp/pull/952/files/27ded86925f799cb6805cfc2244e84f6a4e422c9..4b778fd8a06af328b5b32645e1548c66a3a278c8).

Reply:
I like the naming update 👍🏻